xbmc/pvr/PVRManager.cpp

```diff
@@ -1414,6 +1414,28 @@ bool CPVRManager::IsIdle(void) const
   return true;
 }
 
+bool CPVRManager::CanSystemPowerdown(bool bAskUser /*= true*/) const
```

Comment:
i really dislike multiple return statements in a method, so I use a `bool bReturn` in the pvr code.
please change this code to something like this:
```
bool CPVRManager::CanSystemPowerdown(bool bAskUser /*= true*/) const
{
  bool bReturn(true);
  if (IsStarted())
  {
    if (!m_addons->AllLocalBackendsIdle())
    {
      if (bAskUser)
      {
        // Inform user about PVR being busy. Ask if user wants to powerdown anyway.
        bool bCanceled = false;
        bReturn = CGUIDialogYesNo::ShowAndGetInput(19685, 19686, 0, 0, -1, -1, bCanceled, 10000);
      }
      else
        bRreturn = false; // do not powerdown (busy, but no user interaction requested).
    }
  }
  return bReturn; // powerdown okay.
}
```
(it's less code too ;-))